tests/support/util.tcl

```diff
@@ -639,6 +639,12 @@ proc errorrstat {cmd r} {
     }
 }
 
+proc totalerrorreplies {r} {
+    if {[regexp "\r\ntotal_error_replies:(.*?)\r\n" [$r info stats] _ value]} {
+        set _ $value
+    }
+}
```

Comment:
can't we just do `[s total_error_replies]`?

Reply:
Fixed